.github/workflows/ci.yml

```diff
@@ -48,6 +48,8 @@ jobs:
 
 
       - name: test fx cli
+        env:
+          KUBECONFIG: ${{AKS_KUBECONFIG}}
```

Comment:
If `KUBECONFIG` is too big, should encrypt it and dencript it before using it.
https://help.github.com/en/articles/virtual-environments-for-github-actions